src/ConfigForm.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import './ConfigForm.css'
+
+class ConfigForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      buildId: '',
+      tags: '',
+      platform: 'Windows 11',
+    }
+  }
+
+  handleChange(key, event) {
+    this.setState({[key]: event.target.value,});
+  }
+
+  render() {
+    const {
+      handleConfig
+    } = this.props;
+
+      return (
+        <div className="form" id="form">
+          <form onSubmit={handleConfig}>
+            <label className="form-label">
```

Comment:
Seems a bit unorthodox to put a variety of child elements that aren't related to to the label text into a label element 😅

Reply:
fixed